packages/block-library/src/latest-posts/edit.native.js

```diff
@@ -166,6 +198,31 @@ class LatestPostsEdit extends Component {
 						onChange={ this.onSetDisplayPostDate }
 					/>
 				</PanelBody>
+
+				<PanelBody title={ __( 'Featured image settings' ) }>
+					<ToggleControl
+						label={ __( 'Display featured image' ) }
+						checked={ displayFeaturedImage }
+						onChange={ this.onSetDisplayFeaturedImage }
+					/>
+					{ displayFeaturedImage && (
+						<>
+							<BottomSheetSelectControl
+								label={ __( 'Image alignment' ) }
+								options={ BLOCK_ALIGNMENTS_CONTROLS }
+								onChange={ this.onSetFeaturedImageAlign }
+								value={ featuredImageAlign }
+							/>
```

Comment:
In the future, it would be great if `BlockAlignmentControl` renders this component (i.e. `BottomSheetSelectControl`) underneath, this way we won't need to set up the options and we could use it here.

I was checking [the previous commits](https://github.com/WordPress/gutenberg/pull/39257/commits/9d99ffa0d6850e4127b8e202ead005aaa574a58b) where you introduced `isBottomSheetControl` prop in `BlockAlignmentUI`, and I was wondering if we could follow that approach for this. But, I noticed that it most likely imply a big refactor because we would need to implement the native version of the Block Alignment Toolbar component to provide [here](https://github.com/WordPress/gutenberg/blob/fcd2e769654e272c270f7f725e4087cbe0f3ed8d/packages/block-editor/src/components/block-alignment-control/ui.js#L86) the `BottomSheetSelectControl` component.

Reply:
@fluiddot, thank you for the suggestion! I've gone ahead with the proposed refactor, introducing the `isBottomSheetControl` prop again. As this required a significant number of changes to the `ui.js` file, I created a separate native version in https://github.com/WordPress/gutenberg/pull/39257/commits/19b0e215109e85af346c74bffe2e8de862439626.

Reply:
Great, thank you very much @SiobhyB for tackling this 🙇.

I'd like to clarify that with [my previous comment](https://github.com/WordPress/gutenberg/pull/39257#discussion_r837369643) I didn't want to imply that we had to do the refactor in this PR, but I'm happy to see you sorted it out 😊.  I'll try to take a look as soon as possible at the latest changes.